### ISSUE TITLE
Buses index use

### ISSUE THREAD

**Issue by pabloriera** [pabloriera/lode#11]
Now the input and output buses used by each ode are taken from an index that increments all the time. So when an ode is removed, the buses used by it are left free but are not used anymore if a new ode is added.

It should be good to check unused busses when they are requested.

**Comment by munshkr** [comment]
Right. I'm aware Supercollider's Bus class does this when allocating new buses, maybe we should check out there.
